src/terminal/color.lua

```diff
@@ -0,0 +1,317 @@
+--=============================================================================
+-- text: colors & attributes
+--=============================================================================
+-- Text colors and attributes.
+-- Managing the text color and attributes.
+-- @section textcolor
```

Comment:
for documentation this needs a header, see https://github.com/lunarmodules/terminal.lua/blob/main/src/terminal/input.lua#L1-L14 for an example